modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,26 +78,44 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
 
+|`Import`    |Import configuration options for Argo CD.| `__<object>__`
+a|* `<Name>` - The name of an `ArgoCDExport` resource from which data can be imported.
+  * `<Namespace>` - The namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.
+
 |`ingress`    |Ingress configuration options.| `__<object>__` |
 
 |`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
 
+|`monitoring`    |Defines the workload status monitoring configuration status for your instance.| `__<object>__`
+a|* `<disableMetrics>` - Configure this field to enable or disable the collection of metrics for your instance.
+  * `<enabled>` - Indicates whether the workload status monitoring is enabled for your instance.
+
 |`notifications`    |Notifications controller configuration options.|`__<object>__`
 a|* `<enabled>` - The toggle to start the notifications-controller.
+  * `<env>` -  Environment to set for the notification controller workloads.
   * `<image>` - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
   * `<version>` - The tag to use with the Notifications container image.
+  * `<replicas>` - The number of replicas to be executed for the Notifications controller.
   * `<resources>` - The container compute resources.
   * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
 
 |`repositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.| `__<empty>__` |
 
-|`initialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.| `__<default_Argo_CD_Known_Hosts>__` |
+|`initialSSHKnownHosts`    |Defines Initial SSH Known Hosts data for Argo CD to use upon creation of the cluster for connecting Git repositories by SSH.| `__<default_Argo_CD_Known_Hosts>__`
+a|* `<Excludedefaulthosts>` - Indicates whether you want to add the default list of SSH Known Hosts provided by Argo CD.
+  * `<keys>` - Describes a custom set of SSH Known Hosts that you want to incorporate into your Argo CD server.
 
 |`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |
 
+|`kustomizeVersions`    |Defines a list of `Kustomize` versions that have been configured in the Argo CD Repo Server Container Image.|`__<empty>__`
```

Comment:
```suggestion
|`kustomizeVersions`    |Defines a list of `Kustomize` versions that have been configured in the Argo CD repo server container image.|`__<empty>__`
```

I feel like GitOps is somethimes very inconsistent with capitalization. Should those be really capitalized?

Reply:
I think we can use lowercase in this instance.